ui/core/src/ui-core/utils/hooks/useCrossOriginPopup.tsx

```diff
@@ -7,16 +7,16 @@
  */
 
 import * as React from "react";
-import useInterval from "./useInterval";
+import {useInterval} from "./useInterval";
 
 /** Hook that will show a popup window
  * @beta
  */
-export function usePopup(visible: boolean, url: string | undefined, title: string, width: number, height: number, onClose: () => void) {
+export function useCrossOriginPopup(visible: boolean, url: string | undefined, title: string, width: number, height: number, onClose: () => void) {
```

Comment:
You should probably also set up an onClose listener on the parent window so you can close the child popup if the parent is closed.... ie someone closes browser while your popup is open.